### PR TITLE
Huda/team page

### PR DESCRIPTION
**Pull request by hudahamid**

### CODE REVIEW

File: src/pages/Team/Team.js [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
+    <div >
+      <h1 className="mx-auto  text-5xl  py-16 px-10 lg:px-40">  
+        WE ARE HEALING, NICE TO MEET YOU!
+      </h1>
+      
+      <div className="mt-30 px-4 ">
+
+      <h4 className=" text-4xl  py-14   lg:px-28 pl-8 sm:ml-10 justify-center ">Meet the Team!</h4>
+       {/* maping over the objects */}
+      <div className="flex flex-wrap px-0 pb-24  lg:ml-32   ">
```
Comment (Danah-94) [inline src/pages/Team/Team.js]: suggest to add lg:justify-between and remove lg:ml-32 on line 15

File: src/pages/Team/Team.js [hunk 2]
```diff
@@ -1,7 +1,34 @@
 import React from 'react';
+import constTeam from './ConstTeam';
 
 function Team() {
-  return <h1 className="mx-auto text-center text-7xl">Team</h1>;
+  return (
+    <div >
```
Comment (Danah-94) [inline src/pages/Team/Team.js]: suggest to add the following to line 6:
className="mx-auto lg:max-w-7xl px-10 py-10"

File: src/pages/Team/Team.js [hunk 3]
```diff
@@ -1,7 +1,34 @@
...
+      <h1 className="mx-auto  text-5xl  py-16 px-10 lg:px-40">
```
Comment (Danah-94) [inline src/pages/Team/Team.js]: suggest to add the following to line 7 and remove lg:px-40
className="pt-5 text-3xl md:text-5xl format-normal leading-normal"

**Review by AmenCHW — APPROVED** [review]
looks good, responsiveness need more work, it can be done later

File: src/pages/Team/Team.js (hunk 3 above)
Comment (hudahamid) [reply src/pages/Team/Team.js]: [](url)
result
Danah this is how it look when I apply your suggestion to it

**Pull request closed without merging**